server/modules/doppler-client.js

```diff
@@ -271,6 +271,9 @@ class Doppler {
 
   async putShopifyIntegrationAsync(shopDomain, accessToken) {
     const url = `${baseUrl}/accounts/${this.accountName}/integrations/shopify`;
+    // TODO: Remove logs
+    console.info(shopDomain);
+    console.info(accessToken);
```

Comment:
It will be included in the release? In that case, maybe you can make it more descriptive:

```suggestion
    // TODO: Remove logs
    console.info(`[putShopifyIntegrationAsync] shopDomain: ${shopDomain}; accessToken: ${accessToken}`);
```

Reply:
I think it could be used. I'll use that format, remove the TODO and include it.